config/test.exs

```diff
@@ -6,6 +6,7 @@ config :supavisor,
   api_jwt_secret: "dev",
   metrics_jwt_secret: "dev",
   jwt_claim_validators: %{},
+  proxy_port_session: System.get_env("PROXY_PORT_TRANSACTION", "7653") |> String.to_integer(),
```

Comment:
Apologies, I may be missing something obvious, but is this config used anywhere? Also, the env var does not match the config key, I am not sure if it is intentional.

Reply:
🤦